.github/workflows/upd-xna-client.yml

```diff
@@ -0,0 +1,161 @@
+# To be able use this workflow you must ensure that your repository have enable write and right access for GitHub Actions.
+#
+# GUI Reference to enable access:
+# Repository --> Actions --> General --> Workflow permissions --> Read and write permissions
+#                                                            \--> Allow Github Actions to create and approve pull requests
+
+name: Update XNA CnCNet Client
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *"
+
+env:
+  REPO_NAME: cncnet-client-mod-base
+  CLIENT_PATH: ClientFiles
+  UPDATE_BRANCH_NAME: update-client-binaries
+  COMMIT_MESSAGE: Update client binaries to the latest version
+  PR_TITLE: Update client binaries to the version
+  PR_BODY: This is an automatic pull request to update client binaries triggered from CnCNet/xna-cncnet-client repository.
+
+jobs:
+  publish-update-pr:
+    runs-on: windows-latest
```

Comment:
`windows-latest` is a no-go, specify the fixed version because otherwise we're at risk of stuff breaking

Reply:
Replaced with `windows-2022`